webapp/dev_settings.py

```diff
@@ -1,5 +1,4 @@
-from .settings import *
```

Comment:
I was going with the concept of explicit over implicit imports as PEP8 recommends, I imagined renaming the app would be rare.
There is a good chance we would want to use absolute imports anyway (trying to import `../lib/foo.bar` starts to look messy), and mixing the uses seems untidy to me.